test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -384,20 +385,26 @@ PROTO_TOSTRING(LastStatement, laststat)
 	using LastStatType = LastStatement::LastOneofCase;
 	switch (laststat.last_oneof_case()) {
 	case LastStatType::kExplist:
-		laststat_str = ReturnOptionalExpressionListToString(
-			laststat.explist());
+		if (GetContext().return_is_possible()) {
```

Comment:
Minor: brackets are excess. Here and below.

Reply:
As in my previous PR, I want to keep brackets.